webapp-admin/src/components/SigningLog.js

```diff
@@ -45,33 +53,48 @@ class SigningLog extends Component {
   }
 
   handleItemClick = (index, key) => {
-    this.setState({startRow: 0, endRow: PAGINATION_SIZE});
-    this.props.onItemClick(index, key)
+    var items;
+    if (key === 'models') {
+      items = this.state.filterModels;
+      items[index].selected = !items[index].selected;
+      var filter = items.filter(i => i.selected === true).map(i => i.name).join(',')
+      this.getSigningLogs(this.state.authorityID, 0, filter, this.state.query)
+      this.setState({filterModels: items, filterString: filter});
+    }
   }
 
-  handleRecordsForPage = (startRow, endRow) => {
-    this.setState({startRow: startRow, endRow: endRow});
+  getSigningLogs = (authorityID, offset, filter, serialnumber) => {
+    SigningLogModel.listForAccount(authorityID, offset, filter, serialnumber).then((response) => {
+      var data = JSON.parse(response.body);
+      var message = null;
+      if (!data.success) {
+        message = data.message;
+      }
+      this.setState({logs: data.logs, message: message, totalLogs: data.total});
+    });
+  }
+
+  getSigningLogFilters = (authorityID) => {
```

Comment:
Same here as with `getSigningLogs`, you don't need to pass `authorityID` if it's always from the state

Reply:
fixed